src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--start-day calendar--month-length">
```

Comment:
```suggestion
      <div class="calendar calendar--start-day--fri calendar--month-length--29">
```

Great but add default values for modifiers